docs/modules/ROOT/pages/references/glossary.adoc

```diff
@@ -1,7 +1,72 @@
 = Glossary
 
-https://azure.microsoft.com/de-de/services/kubernetes-service/[AKS]:: Azure Kubernetes Service (AKS)
+////
+You can link these terms using xrefs and their IDs, for example:
+  xref:reference/glossary.adoc#service-instance[Service Instance]
```

Comment:
You can also use `<<service-instance>>` to cross-link on the same page. I suggest doing so for all glossary terms referring to another term.